docs/book/spell-check-custom-words.txt

```diff
@@ -0,0 +1,187 @@
+ABI
```

Comment:
Would it be possible to add this to the gh action so we don't have to maintain copies in every repo?

Reply:
It's definitely possible, I just thought it would slow down development to have to edit 2 repos for docs changes, since there are quite a few tech terms that are unknown to the spell checker.